test/screenshot/lib/screenshot.js

```diff
@@ -17,31 +17,66 @@
 'use strict';
 
 const CbtApi = require('./cbt-api');
+const CbtUserAgent = require('./cbt-user-agent');
+const Duration = require('./duration');
 const ParallelQueue = require('./parallel-queue');
 const Progress = require('./progress');
-const CbtUserAgent = require('./cbt-user-agent');
 
-/** Maximum number of parallel screenshot requests allowed by our CBT plan. */
+/*
+ * Configurable
+ */
+
+/**
+ * Maximum number of parallel screenshot requests allowed by our CBT plan.
+ * @type {number}
+ */
 const API_PARALLEL_REQUEST_LIMIT = 5;
 
-/** How long to wait between polling the API for status changes. */
-const API_POLL_INTERVAL_MS = 1000 * 5;
+/**
+ * Maximum number of times to retry a failed HTTP request.
+ * @type {number}
+ */
+const API_MAX_RETRIES = 5;
 
-/** How long to wait for a single URL to be captured in all browsers. */
-const API_MAX_WAIT_MS = 1000 * 60 * 5;
+/**
+ * How long to wait for a single URL to be captured in all browsers.
+ * @type {!Duration}
+ */
+const API_MAX_WAIT = Duration.minutes(10);
 
-/** Maximum number of times to retry a failed HTTP request. */
-const API_MAX_RETRIES = 5;
+/**
+ * How long to wait between polling the API for status changes.
+ * @type {!Duration}
+ */
+const API_POLL_INTERVAL = Duration.seconds(5);
 
-/** Map of URLs to `Progress` objects. */
-const progressMap = new Map();
+/**
+ * How long to wait between retrying failed requests.
+ * @type {!Duration}
+ */
+const API_RETRY_WAIT = Duration.seconds(10);
+
+/*
+ * Non-configurable
+ */
+
+/** @type {number} */ const API_MAX_WAIT_MS = API_MAX_WAIT.toMillis();
+/** @type {number} */ const API_POLL_INTERVAL_MS = API_POLL_INTERVAL.toMillis();
+/** @type {string} */ const API_POLL_INTERVAL_HUMAN = API_POLL_INTERVAL.toHuman();
+/** @type {number} */ const API_RETRY_WAIT_MS = API_RETRY_WAIT.toMillis();
+/** @type {string} */ const API_RETRY_WAIT_HUMAN = API_RETRY_WAIT.toHuman();
 
+/** @type {!CbtApi} */
 const cbtApi = new CbtApi();
 
+/** @type {!ParallelQueue<string>} */
+const requestQueue = new ParallelQueue({maxParallels: API_PARALLEL_REQUEST_LIMIT});
+
 /**
- * @type {!ParallelQueue<string>}
+ * Map of URLs to `Progress` objects.
+ * @type {!Map<string, !Progress>}
  */
-const requestQueue = new ParallelQueue({maxParallels: API_PARALLEL_REQUEST_LIMIT});
+const progressMap = new Map();
```

Comment:
Why did this change?

Reply:
I added a `@type` annotation to `progressMap`'s JSDoc, and I thought the code looked better cosmetically with the one-line JSDocs first, so I reordered `requestQueue` and `progressMap` 😄